packages/web-runtime/src/store/config.js

```diff
@@ -69,7 +69,7 @@ const actions = {
 
 const mutations = {
   LOAD_CONFIG(state, config) {
-    state.server = config.server.endsWith('/') ? config.server : config.server + '/'
```

Comment:
Why did you get rid of this? I wanted to enforce here, that we can rely on a) always having a trailing slash or b) never having a trailing slash. I don't care which one, but getting rid of this leaves potential for faulty configuration from the sysadmin.